docs/app/pages/components/components-sections/notifications/notifications/notifications.component.html

```diff
@@ -0,0 +1,61 @@
+<ux-notification-list></ux-notification-list>
+
+<button (click)="notificationService.showNotification(myNotification, {})">Show Notification</button>
+<button (click)="notificationService.dismissAllNotifications()">Dismiss Notifications</button>
+
+<hr>
+
+<p>Notifications can be used to inform the user of occurring events or to alert the user of something that may require their
+    attention. Notifications will appear in the top right corner of the browser window and can be dismissed by the user by
+    clicking the close button.</p>
```

Comment:
Position is configurable, so not necessarily top right. Also by default, notifications dismiss after 4 seconds (also configurable).

Reply:
Documented that position and duration are configurable.